react_native/src/screens/root_screens/RootStackScreenComponent.js

```diff
@@ -123,7 +123,7 @@ function RootStackScreenComponent() {
     }, []);
 
     if (loginState.isLoading) {
-        //dispatch({type: 'LOGOUT'}); //for now it does not loop forever, we'll see what happens
+        dispatch({type: 'LOGOUT'}); //for now it does not loop forever, we'll see what happens
```

Comment:
We have to keep an eye on this after the merge, this is inconsistent and the app shows the splash screen for a split second when it opens up and retrieves the token if we keep this hardcoded here